### ISSUE TITLE
Triage MultiDynamoDBStoreTest failure in Travis CI

### ISSUE THREAD

**Issue by amcp** [awslabs/dynamodb-titan-storage-backend#97]
Recent builds:
https://travis-ci.org/awslabs/dynamodb-titan-storage-backend/jobs/226287931
https://travis-ci.org/awslabs/dynamodb-titan-storage-backend/jobs/226377196

Travis CI output at the end of the file. Usually takes around 14 minutes to run. These tests pass when run using the instructions at the end of the README on EC2 m4.10xlarge instances.

```bash
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running com.amazon.titan.diskstorage.dynamodb.MultiDynamoDBStoreTest
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
The build has been terminated
```

**Comment by amcp** [comment]
@F2006 This test is the one causing the errored emails I think.
https://travis-ci.org/awslabs/dynamodb-titan-storage-backend/builds/226407287
shows "Job errored" status, while for the GraphTest ones it only shows "Job failed". Currently this test times out for lack of output. Can you make a PR that causes something to be printed every five minutes or so to the console?

**Comment by F2006** [comment]
@amcp pull request (https://github.com/awslabs/dynamodb-titan-storage-backend/pull/102) made, will monitor the build and see how it goes: https://travis-ci.org/awslabs/dynamodb-titan-storage-backend/builds/226481447

**Comment by amcp** [comment]
@F2006 thank you. I reviewed, and now I will sleep, so I will look at your updates in the morning.

**Comment by amcp** [comment]
Probably fixed in #102, will merge and then enable the MultiStoreTest again

**Comment by amcp** [comment]
https://travis-ci.org/awslabs/dynamodb-titan-storage-backend/jobs/227081845#L4552

Heart is beating!
